backend/back/src/types/jobParams.ts

```diff
@@ -317,3 +318,17 @@ export class SlackCreateMessageParams extends UniqueJobParams {
 	@IsString()
 	text: string;
 }
+
+export class DiscordOptionalGuildIdParams extends UniqueJobParams {
+	@IsOptional()
+	@IsString()
+	@Matches(/^\d{17,19}$/)
+	guildId: string;
+}
+
+export class DiscordGuildIdParams extends UniqueJobParams {
+	@IsOptional()
+	@IsString()
+	@Matches(/^\d{17,19}$/)
+	guildId: string;
+}
```

Comment:
Those are the same, I guess the second one shouldn't have @IsOptional()

Reply:
yes you are right